homeassistant/components/wled/light.py

```diff
@@ -400,6 +419,7 @@ def async_update_segments(
     coordinator: WLEDDataUpdateCoordinator,
     current: Dict[int, WLEDSegmentLight],
     async_add_entities,
+    title_base=None,
```

Comment:
Don't pass this along constantly, please re-use the coordinator that is already available everywhere.